src/libhirte/ini/config.c

```diff
@@ -5,21 +5,33 @@
 #include "ini.h"
 #include "libhirte/common/common.h"
 
-int match(const char *section, const char *check_section, const char *name, const char *check_name) {
-        if (streq(section, check_section) == 0 && streq(name, check_name) == 0) {
-                return EXIT_SUCCESS;
-        }
-        return EXIT_FAILURE;
+typedef struct {
+        char *key;
+        char *value;
+} keyValue;
```

Comment:
We've been following the systemd coding style, and there a struct type starts with a capital letter, so ```KeyValue```.